dbms/src/main/java/org/polypheny/db/processing/AbstractQueryProcessor.java

```diff
@@ -295,6 +346,25 @@ protected PolyphenyDbSignature prepareQuery( RelRoot logicalRoot, RelDataType pa
                 if ( isAnalyze ) {
                     statement.getDuration().stop( "Implementation Caching" );
                 }
+
+                //TODO @Cedric this produces an error causing several checks to fail. Please investigate
+                //needed for row results
+
+                //final Enumerable enumerable = signature.enumerable( statement.getDataContext() );
+                //Iterator<Object> iterator = enumerable.iterator();
```

Comment:
Is this something that needs to be fixed in a different branch?

Reply:
This will be indeed fixed in Cedric branch: Universal routing.